lib/stm32/l0/rcc.c

```diff
@@ -336,6 +336,19 @@ void rcc_set_pll_divider(uint32_t factor)
 	RCC_CFGR = reg | (factor << RCC_CFGR_PLLDIV_SHIFT);
 }
 
+/**
+ * Set the pll source.
+ * @param pllsrc RCC_CFGR_PLLSRC_HSI16_CLK or RCC_CFGR_PLLSRC_HSE_CLK
+ */
+void rcc_set_pll_source(uint32_t pllsrc)
+{
+	uint32_t reg32;
+
+	reg32 = RCC_CFGR;
+	reg32 &= ~(RCC_CFGR_PLLSRC_HSE_CLK << 16);
+	RCC_CFGR = (reg32 | (pllsrc<<16));
```

Comment:
Could you consider replacing the `16` by something like `RCC_CFGR_PLLSRC_SHIFT`?

Reply:
I could, and I largely left it there to see if anyone was even reading :)  It's a copy of the existing L1 code, which would then also need to be done, and we don't tend to define single bits.  It's only a single bit on L0/L1, but a range on others, which is why the bit is defined unshifted here, vs shifted, as most single bits are.  joy and confusion